### PR TITLE
Merge  v0.4.3

### PR DESCRIPTION
**Pull request by mprevide**

### CODE REVIEW

File: travis/deploy-gh-pages.sh [hunk 1]
```diff
@@ -1,28 +1,24 @@
...
-    git push --force http://${GITHUB_TOKEN}:x-oauth-basic@github.com/${TRAVIS_REPO_SLUG} temp-pages:gh-pages
+echo "Don't worry about the above errors"
 
-    git checkout . 
-    git clean -fd
-    git checkout ${TRAVIS_BRANCH} origin-gh-pages/${TRAVIS_BRANCH} 
-fi
+git add apiary_${VERSION}.html
+git commit -m 'Updating gh-pages'
+git push http://${GITHUB_TOKEN}:x-oauth-basic@github.com/${TRAVIS_REPO_SLUG} gh-pages
```
Comment (thiagodpf) [inline travis/deploy-gh-pages.sh:21]: Would it be okay to use HTTPS?

File: travis/deploy-gh-pages.sh [hunk 2]
```diff
@@ -1,28 +1,24 @@
...
-    export VERSION="${TRAVIS_BRANCH}"
-    fi
-
-    docker run --volume $(pwd):/temp:Z dojot/aglio -i /temp/apiary.apib -o - > ./apiary_${VERSION}.html
+docker run --volume $(pwd):/temp:Z dojot/aglio -i /temp/apiary.apib -o - > ./apiary_${VERSION}.html
 
-    git add apiary_${VERSION}.html
-    git commit -m 'Updating gh-pages' 
-    git push --force http://${GITHUB_TOKEN}:x-oauth-basic@github.com/${TRAVIS_REPO_SLUG} temp-pages:gh-pages
+echo "Don't worry about the above errors"
```
Comment (thiagodpf) [inline travis/deploy-gh-pages.sh:17]: If you redirect `stderr` to somewhere other than the terminal, you may not need this message...

```bash
docker run --volume $(pwd):/temp:Z dojot/aglio \
    -i /temp/apiary.apib -o - \
    > ./apiary_${VERSION}.html \
    2> /dev/null
```

**Review by thiagodpf — APPROVED** [review]
It is very good for me. I left some suggestions, but they are not necessary, so on my part, it's already approved!

**Review by eduardogmisiuk — APPROVED** [review]
LGTM!

**Pull request merged**